scripts/readme.md

```diff
@@ -0,0 +1,11 @@
+## Scripts for data processing
```

Comment:
Thanks for the documentation!
I think there will be many other scripting tasks.
Please move this to a directory named specifically for amazon-scraping vs 'scripts'